doc/source/security/selinux.rst

```diff
@@ -0,0 +1,136 @@
+.. _selinux:
+
+===========================================
+Enabling SELinux 
+===========================================
+
+SELinux helps protects the user's home directory data. SELinux provides the following:
+
+* Prevents unauthorized users from exploiting the system
+
+* Allows authorized users to access files 
+
+* Used as a role-based access control system
+
+For more information, see `Percona Server and SELinux <https://www.percona.com/doc/percona-server/LATEST/security/selinux.html>`_
+
+Red Hat and CentOS distributes profiles to extend the SELinux policy module for mysqld. The following profiles distributed along with the extended default ``mysqld`` profile allow PXC-specific operations:
```

Comment:
1. Naming:
SELinux => policy modules
AppArmor => security profiles

2. we do not distribute pxc module along with the extended default MySQL module. For PS, we do not distribute SELinux module, as the default one distributed with the OS is enough. For PXC we distribute pxc SELinux module, which extends rules defined by the default MySQL SELinux module.

3. If we talk about AppArmor, OS does not provide any profile, so for PS we distribute PS profile and for PXC we distribute PXC profile (which is the extended version of PS profile) + SST script profile.

Reply:
Updated.

Reply:
Did a search and replace for profiles.